### PR TITLE
Updated membrane movement

### LINKED ISSUE #1208: Check that the wobble on the more rigid walls / membranes is reduced
https://community.revolutionarygamesstudio.com/t/0-5-0-public-beta-godot-version/2617/4?u=hhyyrylainen

### PR DESCRIPTION
**Pull request by icedjuro**
Closes #1208

### CODE REVIEW

**Comment by buckly90** [comment]
Can confirm that this results in significantly less wobble on all the cell walls.

**Comment by CoreyHendrey** [comment]
Just an idea, I wonder if we could mix Rigidity in here somewhere?

File: src/microbe_stage/Membrane.cs [hunk 1]
```diff
@@ -542,6 +542,8 @@ private void BuildMesh()
         float multiplier = 2.0f * Mathf.Pi;
         var center = new Vector2(0.5f, 0.5f);
 
+        movementWigglyNess = Type.MovementFactor;
```
Comment (hhyyrylainen) [inline src/microbe_stage/Membrane.cs:545]: This is wrong, the wigglyness shouldn't be set from the movement factor. Movement factor is only how much the base movement speed is modified.
A new property should be added to Type that is the movementwigglyness value and is set from the json to make things configurable.

File: simulation_parameters/microbe_stage/membranes.json [hunk 2]
```diff
@@ -28,6 +30,7 @@
         "normalTexture": "res://assets/textures/CellWallGradient.png",
         "damagedTexture": "res://assets/textures/CellWallGradientDamaged.png",
         "movementFactor": 0.8,
+        "movementWigglyness": 0.8,
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/membranes.json:33]: I think this should be set to the same value as chitin, as the cell wall is not meant to be so wiggly

File: simulation_parameters/microbe_stage/membranes.json [hunk 3]
```diff
@@ -27,7 +29,8 @@
         "cellWall": true,
         "normalTexture": "res://assets/textures/CellWallGradient.png",
         "damagedTexture": "res://assets/textures/CellWallGradientDamaged.png",
-        "movementFactor": 0.8,
+        "movementFactor": 0.4,
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/membranes.json:32]: The movement factor has balancing implications and I don't think it should be included in this PR
This is why I wanted the movement wigglyness to be a separate thing

**Comment by hhyyrylainen** [comment]
JSONlint wants one change for this.

**Pull request merged**